airflow/providers/common/sql/hooks/sql.py

```diff
@@ -372,6 +372,15 @@ def run(
         else:
             return results
 
+    def _make_serializable(self, result: Any) -> Any:
+        """Ensure the data returned from a SQL command is JSON-serializable.
+
+        This method is intended to be overridden by subclasses of the
+        `DbApiHook`. Its purpose is to transform the result of a SQL command
+        into a JSON-serializable format.
```

Comment:
What should happen if this is not overridden? If we don't want the user do not override it, should we add an exception here? If it's ok that we return the original input as output, could we add this behavior to the docstring?

Reply:
In most cases, this won't be overidden, and indeed the original input is returned as output. For what I know, the only cases where they are custom Row objects are in Databricks and PyODBC. I'll edit the docstring.